index.html

```diff
@@ -0,0 +1,258 @@
+          <span class="bar"></span>
+        </div>
+        <!-- <ul id="mobile-nav-menu">
+              <li class="mobile-nav-link"><a href="#works">Portfolio</a></li>
+              <li class="mobile-nav-link"><a href="#about-myself">About</a></li>
+              <li class="mobile-nav-link"><a href="#form-section">Contact</a></li>
+            </ul> -->
+      </div>
+    </nav>
+  </header>
+
+  <section id="headline">
+    <div id="headline-content">
+      <h1 class="header">I'm Anuj, <br> Glad to see you!</h1>
+      <p class="portfolio-description">I’m a software developer! I can help you build a product , feature or website
+        Look
+        through some of my work
+        and experience. If you like what you see and have a project you need coded, don’t hestiate to contact me!
+      </p>
+      <h2 class="lets-connect">LET’S CONNECT</h2>
+      <ul class="social-media-links">
+        <li> <a href="#" target="_blank" rel="noopener" class="social-media-link"><i class="fa-twitter">
+              <img src="images/twitter.svg" alt="twitter">
+            </i> </a> </li>
+        <li> <a href="#" target="_blank" rel="noopener" class="social-media-link"><i class=" fa-linkedin"></i>
+            <img src="images/in.png" alt="linkedin">
+          </a> </li>
+        <li> <a href="#" target="_blank" rel="noopener" class="social-media-link"><i class="fa-github">
+              <img src="images/medium.svg" alt="medium">
+            </i>
+          </a></li>
+        <li> <a href="#" target="_blank" rel="noopener" class="social-media-link">
+
+            <i class="fa-brands fa-hackerrank fa-2x">
+
+              <img src="images/github.svg" alt="github">
+            </i> </a></li>
+
+        <li> <a href="#" target="_blank" rel="noopener" class="social-media-link">
+
+            <i class="fa-brands fa-hackerrank fa-2x">
+
+              <img src="images/angellist.svg" alt="angellist">
+            </i> </a></li>
+      </ul>
+    </div>
+  </section>
+
+  <!-- The "Works" section - Projects I've worked successfully. -->
+  <section class="works" id="works">
+    <section class="work">
+      <div class="right-block right-block-one">
+        <img src="/images//Portfolio1.png" alt="Tonic Project" />
+      </div>
+      <div class="left-block left-block-one">
+        <h3 class="work-title">Tonic</h3>
+        <ul class="work-stack">
+          <li>Canopy</li>
+          <li class="dot"></li>
+          <li>Backend Dev</li>
+          <li class="dot"></li>
+          <li>2015</li>
+        </ul>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <div class="action">
+          <button type="button" data-id="1">See Project</button>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] For semantic reasons, for each work card you can use other element tags like `<article>` instead of `<section>` tag since you have a `<section>` tag as the container of the works section.